src/utrition/src/components/imageupload/imageupload.js

```diff
@@ -14,11 +17,81 @@ const ImageUpload = () => {
     upload_type: "image",
   };
 
+  // New header
+  const headers_confirm = {
+    upload_type: "confirm",
+  };
+
+  
   const handleClick = (event) => {
     fileInput.current.click();
   };
 
+  // Doesn't send any response, set food to 'correct', close confirmation
+  function handleCorrectData() {
+    setCorrectFood(true);
+    setShowConfirmationDialog(false);
+  }
+
+  // Sends response to /upload, with new header;
+  // set food to 'incorrect', close confirmation
+  // populate and show error message
+  async function handleIncorrectData() {
+    setCorrectFood(false);
+    const formData = new FormData();
+    formData.append('index', null);
+    await axios({
+      method: "POST",
+      headers: headers_confirm,
+      url: "/upload",
+      data: formData,
+    })
+      .then()
+      .catch((error) => console.log(error));
+      setResponseData({
+        error_msg: "Please try again or use a different image."
+      })
+      setShowConfirmationDialog(false);
+  }
+
+  // Shows confirmation if in the correct state;
+  // user will either deem the output correct/incorrect;
+  // handled 2 different ways
```

Comment:
Conditionally renders confirmation menu to prompt user to confirm their classification